### PR TITLE
Team Pattern Created

### LINKED ISSUE #117: Block Patterns - Team
**Description**

🛠️ The goal of this issue is to track the implementation of the "Team" pattern.
🎨 [Figma](https://www.figma.com/file/AlYr03vh4dVimwYwQkTdf6/Twenty-Twenty-Four?type=design&node-id=565%3A10409&mode=design&t=LHSPHKUrPq556l59-1)

Team

### PR DESCRIPTION
**Pull request by devmuhib009**
Updated Team Pattern

**Description**

Just created and updated team pattern

### CODE REVIEW

**Review by maneshtimilsina — COMMENTED** [review]
It will be better to remove the ID and class of the images.

**Comment by carolinan** [comment]
This PR replaces https://github.com/WordPress/twentytwentyfour/pull/24

**Comment by huzaifaalmesbah** [comment]
Related #117

**Comment by carolinan** [comment]
We need to include the original images without the rounded corners and only change the radius with CSS.

**Comment by devmuhib009** [comment]
When I save images from Figma file, it comes with rounded corner. However, I will recheck and add original images with CSS code to make them rounded.

**Comment by carolinan** [comment]
To clarify I did not mean adding any CSS to the theme. I think the rounded style should be used, like in this PR.

File: patterns/team.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+<!-- wp:paragraph {"align":"center","fontSize":"small"} -->
+<p class="has-text-align-center has-small-font-size"><?php echo esc_html_x( 'Engineering Manager', 'Team Member Designation', 'twentytwentyfour' ); ?></p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"layout":{"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:image {"align":"center","scale":"cover","sizeSlug":"full","linkDestination":"none","style":{"border":{"radius":"16px"}}} -->
+<figure class="wp-block-image aligncenter size-full has-custom-border"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/team-member-3.png" alt="" style="border-radius:16px;object-fit:cover"/></figure>
```
Comment (MaggieCabrera) [inline patterns/team.php]: This image already exists in the assets folder named avatar_4.jpg and it's used in the sidebar pattern. I'm happy with renaming it there if you want, but let's not have duplicated assets in any case

File: patterns/team.php [hunk 2]
```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Title: Team
+ * Slug: twentytwentyfour/team
+ * Categories: team
```
Comment (MaggieCabrera) [inline patterns/team.php]: This category doesn't exist, let's use featured and text instead

File: patterns/team.php (hunk 1 above)
Comment (devmuhib009) [reply patterns/team.php]: Sure. You can remove the duplicate one.

File: patterns/team.php [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+<!-- wp:paragraph {"align":"center","fontSize":"small"} -->
+<p class="has-text-align-center has-small-font-size"><?php echo esc_html_x( 'Founder, CEO & Architect', 'Team Member Designation', 'twentytwentyfour' ); ?></p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"layout":{"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:image {"align":"center","scale":"cover","sizeSlug":"full","linkDestination":"none","style":{"border":{"radius":"16px"}}} -->
+<figure class="wp-block-image aligncenter size-full has-custom-border"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/team-member-2.png" alt="" style="border-radius:16px;object-fit:cover"/></figure>
```
Comment (MaggieCabrera) [inline patterns/team.php]: let's make use of aspect ratio here and use the square one so all images are the same size

File: patterns/team.php (hunk 2 above)
Comment (devmuhib009) [reply patterns/team.php]: Ok

**Review by MaggieCabrera — CHANGES_REQUESTED** [review]
This is generally looking great, just a few comments and it should be good to go

File: patterns/team.php (hunk 3 above)
Comment (devmuhib009) [reply patterns/team.php]: Ok

**Comment by devmuhib009** [comment]
@MaggieCabrera I will update the changes and inform you.

**Review by maneshtimilsina — CHANGES_REQUESTED** [review]
- Space/padding at the top and bottom of the section is missing. Please check Figma and update it accordingly. Let's use the spacing presets for paddings where needed.
- Make it align full instead of wide
- Font size of the Subheader is larger than it is set in Figma
- You are using margin-top 64px for the row used in the pattern. Instead of arbitrary values, please use the spacing presets.
- I would suggest using columns instead of row to display team members in a grid.
- Replace current images with large size images and check once.

File: patterns/team.php [hunk 4]
```diff
@@ -2,84 +2,91 @@
...
-<div class="wp-block-group"><!-- wp:paragraph {"align":"center","style":{"typography":{"fontStyle":"normal","fontWeight":"600"}},"fontSize":"medium"} -->
-<p class="has-text-align-center has-medium-font-size" style="font-style:normal;font-weight:600"><?php echo esc_html_x( 'Francesca Piovani', 'Team Member Name', 'twentytwentyfour' ); ?></p>
+<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|0"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:paragraph {"align":"center","fontSize":"small"} -->
+<p class="has-text-align-center has-small-font-size"><strong><?php echo esc_html__( 'Francesca Piovani', 'twentytwentyfour' ); ?></strong></p>
 <!-- /wp:paragraph -->
 
 <!-- wp:paragraph {"align":"center","fontSize":"small"} -->
-<p class="has-text-align-center has-small-font-size"><?php echo esc_html_x( 'Founder, CEO & Architect', 'Team Member Designation', 'twentytwentyfour' ); ?></p>
+<p class="has-text-align-center has-small-font-size"><?php echo esc_html__( 'Founder, CEO &amp; Architect', 'twentytwentyfour' ); ?></p>
```
Comment (MaggieCabrera) [inline patterns/team.php]: why did you remove the description of the translations?

Comment (devmuhib009) [reply patterns/team.php]: I did some structure change, so previous translation and other things removed automatically. So, I did all manually again. In most of the files I noticed there is esc_html__ being used. So, I thought both are ok.

**Comment by huzaifaalmesbah** [comment]
Looks Good

**Comment by devmuhib009** [comment]
Thank you :)

**Comment by huzaifaalmesbah** [comment]
Add image alt text and ready for translation `esc_attr__`

**Comment by devmuhib009** [comment]
Can you please give an example or where I can find this? Please give a file name where I will find sample of alt. Thanks

**Comment by huzaifaalmesbah** [comment]
you can check for example [esc_attr__](https://github.com/WordPress/twentytwentyfour/pull/192/commits/4109d5745bfae8f1ed55f877e25c41d9abf0da2e)

**Comment by devmuhib009** [comment]
ok

**Comment by devmuhib009** [comment]
> you can check for example [esc_attr__](https://github.com/WordPress/twentytwentyfour/pull/192/commits/4109d5745bfae8f1ed55f877e25c41d9abf0da2e)

Added the alt tag. Thanks for the suggestion.

**Review by huzaifaalmesbah — COMMENTED** [review]
Looks Good

**Review by MaggieCabrera — APPROVED** [review]
I made a couple of changes, this is good to go, thank you for your work!

**Comment by devmuhib009** [comment]
Thank you! Learned lots of things from this specific contribution. ❤️❤️❤️
